plugins/clock-simple/manager.py

```
@@ -456,7 +455,7 @@ def display(self, force_clear: bool = False) -> None:
                     color=(255, 0, 0)
                 )
                 self.display_manager.update_display()
-            except:
+            except Exception:
                 pass  # If display fails, don't crash
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)